### DISCUSSION TITLE
Env selection when using runner on a folder

### DISCUSSION THREAD

**Discussion by adri1g** [usebruno/bruno#1467] [Ideas]
Hello,
Combo list for environment selection is not displayed when launching a runner on a folder. It appears only when I open a request.
Could be useful to fix this point :)
Regards,
Adrien

**Comment by helloanoop** [comment]
@adri1g I am able to see the environment selector as seen below in the screenshot. I am on the version `v1.6.1`.
Can you share a screenshot of the issue ?

![image](https://github.com/usebruno/bruno/assets/9350733/7ce19eab-f0a8-4a99-af0c-9845098fe1de)

**Reply by adri1g** [reply]
Ok but you cannot select the env before you open a runner tab so all the requests fail before you run again.
![image](https://github.com/usebruno/bruno/assets/90836584/23929be6-095e-45c7-8181-4dd43d230e81)

**Reply by helloanoop** [reply]
Yup. Makes sense now!
